### PR TITLE
Build errors and coding in header Backup book

### PR DESCRIPTION
**Pull request by swopebe**

### CODE REVIEW

File: business_continuity/backup_restore/backup_validate.adoc [hunk 1]
```diff
@@ -36,13 +36,13 @@ The following templates check the pod status for the backup component and depend
 +
 * `backup-storage-location-available` template checks if a `BackupStorageLocation.velero.io` resource is created and if the status value is `Available`. This implies that the connection to the backup storage is valid. 
 
-- *BackupSchedule collision validation*
+- *`BackupSchedule` collision validation*
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_validate.adoc:39]: If you are using these as they are coded, we need to reflect that. When translated, the other party would not know this is part of a code function and would not recognize this as a word. That is why we either use the general term "backup schedule" or we code it as it is in the product. 

No hard rule about which you choose, other than have it reflect the purpose of the content.

File: release_notes/acm_whats_new.adoc [hunk 2]
```diff
@@ -118,7 +118,7 @@ See link:../governance/grc_intro.adoc#governance[Governance] to learn more about
 
 * You can now customize the OADP version by setting an annotation on your `MultiClusterHub` resource. See link:../business_continuity/backup_restore/backup_install.adoc#installing-custom-oadp[Installing a custom OADP version].
 
-* You can now temporarily pause the `BackupSchedule` resource instead of deleting it. By using the `BackupSchedule` `paused` property on the backup hub cluster, you can avoid a backup collision. See link:../business_continuity/backup_restore/backup_schedule.adoc#avoid-backup-collision[Avoiding backup collisions].
+* You can now temporarily pause the `BackupSchedule` resource instead of deleting it. By using the `BackupSchedule` `paused` property on the backup hub cluster, you can avoid a backup collision. See link:../business_continuity/backup_restore/backup_schedule.adoc#prevent-backup-collision[Preventing backup collisions].
```
Comment (swopebe) [inline release_notes/acm_whats_new.adoc:122]: This is what caused the error and I think you changed it to "prevent" but please check.

File: business_continuity/backup_restore/backup_validate.adoc [hunk 3]
```diff
@@ -36,13 +36,13 @@ The following templates check the pod status for the backup component and depend
...
 +
 * `acm-backup-clusters-collision-report` template verifies that the status is not `BackupCollision`, if a `BackupSchedule.cluster.open-cluster-management.io` exists on the current hub cluster. This verifies that the current hub cluster is not in collision with any other hub cluster when you write backup data to the storage location.
 +
-For a definition of the `BackupCollision`, see xref:../backup_restore/backup_schedule.adoc#avoid-backup-collision[Avoiding backup collisions].
+For a definition of the `BackupCollision`, see xref:../backup_restore/backup_schedule.adoc#prevent-backup-collision[Preventing backup collisions].
```
Comment (swopebe) [inline business_continuity/backup_restore/backup_validate.adoc:43]: This is what caused the error and I think you changed it to "prevent" but please check.

Comment (swopebe) [reply business_continuity/backup_restore/backup_validate.adoc:43]: link passed local test.

**Review by jc-berger — APPROVED** [review]
/lgtm

**Pull request merged**